.cicd/scripts/srw_build.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
Although it becomes clear after one reads through this script, Including a brief comment at the top describing what it does would be helpful.

Reply:
Great, thanks!